template.yaml

```diff
@@ -1,8 +1,8 @@
-AWSTemplateFormatVersion: '2010-09-09'
+AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
 Description: >
   identity-service
-  
+
   Sample SAM Template for identity-service
 
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
```

Comment:
@MehulKChaudhari Can you remove this template.yaml changes we don't need that

Reply:
There is no change there ig. The prettier has only removed the extra space on that line.